core/src/test/scala/org/apache/spark/sql/delta/stats/StatsCollectionSuite.scala

```diff
@@ -131,7 +131,8 @@ class StatsCollectionSuite
         val statsDf = statsDF(deltaLog)
         assert(statsDf.where('numRecords.isNotNull).count() > 0)
         // scalastyle:off line.size.limit
-        val expectedStats = Seq(Row(3, Row(10, 20), Row(19, 29)), Row(4, Row(12, 22), Row(17, 27)), Row(3, Row(11, 21), Row(18, 28)))
+        // TODO: Fix test on migrating to Spark 3.4
```

Comment:
can you mention what needs to be fixed? should we create a github issue for this with a bit more detail and tagged with the 2.4.0 milestone?

Reply:
Same as above just reverting https://github.com/delta-io/delta/commit/529313d61a0d998b66b6fa0cad7d35e971edfb94 for tests to pass. Fix is in 3.3.1.